prisma/schema.prisma

```diff
@@ -522,3 +524,12 @@ model VerificationRequest {
 
   @@map(name: "verification_requests")
 }
+
+model Bookmark {
+  user      User     @relation(fields: [userId], references: [id])
+  userId    Int
+  item      Item     @relation(fields: [itemId], references: [id])
+  itemId    Int
+  createdAt DateTime @default(now()) @map(name: "created_at")
+  @@id([userId, itemId])
```

Comment:
I think I don't need an additional index on `userId` to speed up querying user bookmarks since it's part of the primary index.

Reply:
Yeah afaik that index can be used without specifying an itemId. You probably want an index on createdAt though for sorting.

Reply:
Ah, right. Added an index